core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -337,14 +337,15 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
               // determine which functions to use for the various part of the response
               def bucketOptionsFuture(): Future[Option[WorkspaceBucketOptions]] =
                 if (options.contains("bucketOptions")) {
-                  wsmContext.azureCloudContext match {
+                  wsmContext.googleProjectId match {
```

Comment:
Before this keyed off of `azureCloudContext`, returning bucket details if None. I have swapped the check so that we return bucket details if `googleProjectId` is not None, to handle the case of them both being None (plus it just makes more sense!).